documentation/demopages/nl-design-system/common/mixins/_alerts.scss

```diff
@@ -0,0 +1,70 @@
+@mixin alerts {
```

Comment:
Algemeen puntje: Ik ben benieuwd waarom deze mixins op deze manier en plek zijn gedefinieerd. En waarom gebruiken we hier niet de design tokens zoals gedefinieerd in alert.tokens.json?

Reply:
Ik heb de opzet van de demopages/common/style.scss bestand overgenomen en opgesplitst in losse bestanden om ze 1 voor 1 over te zetten of om te zetten in bestaande design system oplossingen. De mixin aanpak is zodat ik een ander base class kan gebruiken naast de bestaande opzet. Zo voorkom ik ook duplicaat code in storybook tijdens ontwikkelen. (Ik zag de originele RVO styling de NL DS styling overschrijven in Storybook terwijl de RVO code niet werd aangeroepen.)

Reply:
Het lijkt er nu wel op dat ik de mixins niet meer nodig heb. Dus heb ik het behoorlijk terug kunnen brengen.